startup/01-bluesky.py

```diff
@@ -15,7 +16,7 @@
 from bluesky.utils import PersistentDict
 
 # runengine_metadata_dir = appdirs.user_data_dir(appname="bluesky") / Path("runengine-metadata")
-runengine_metadata_dir = Path(f"/GPFS/CENTRAL/xf17id1/skinnerProjectsBackup/bnlpx_config/{BEAMLINE_ID}_bluesky_config/")
+runengine_metadata_dir = Path(f"/nsls2/data/{BEAMLINE_ID}/shared/config/{BEAMLINE_ID}_bluesky_config/")
```

Comment:
I think we need to recursively create this dir in `.ci/bl-specific.sh` (instead of the previously-created dir `/GPFS/...`).

Reply:
done in https://github.com/NSLS-II-FMX/profile_collection/pull/17/commits/34f96dca4688293b2e0c9127f157722382008def